community/modules/remote-desktop/chrome-remote-desktop/variables.tf

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "Project in which Google Cloud resources will be created"
+  type        = string
+}
+
+variable "deployment_name" {
+  description = "HPC Toolkit deployment name. Cloud resource names will include this value."
+  type        = string
+  default     = "chrome-remote-desktop"
+}
+
+variable "region" {
+  description = "Default region for creating resources"
+  type        = string
+}
+
+variable "zone" {
+  description = "Default zone for creating resources"
+  type        = string
+}
+
+variable "name_prefix" {
+  description = "Name prefix for remote desktop hostnames"
+  type        = string
+  default     = null
+}
+
+variable "instance_count" {
```

Comment:
here, it is important to allow people to set this to zero. In which case, we could still have a variable with the startup script generated. That could be useful when using this module in conjunction to schedulers. For instance, one could create a partition in Slurm with schedmd-v5-slurm-22-05-6-ubuntu-2004-lts image and run this on top of that image... :) 

In principle, this all works by design, so the only thing to do with this comment is to eventually test it with the above mentioned image (project schedmd-slurm-public).

Reply:
easy to test by setting the instance_image variable below with project=schedmd-slurm-public and family=schedmd-v5-slurm-22-05-6-ubuntu-2004-lts

Reply:
I have opened a task to test this in the future.